### PR TITLE
TF-1149 Fix bug Swipe right when opening mail leads to a random screen

### PR DESCRIPTION
**Pull request by sherlockvn**
#1149 

https://user-images.githubusercontent.com/43041967/201024830-001f8dd5-534a-46e0-b2c2-2dc35bf919d8.mp4

https://user-images.githubusercontent.com/43041967/202349733-c0a45af9-5311-42c6-a645-e1d33fd7bcc7.mov

### CODE REVIEW

File: lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart [hunk 1]
```diff
@@ -5,16 +5,49 @@ import 'package:get/get.dart';
...
+import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart';
 import 'package:tmail_ui_user/features/mailbox_dashboard/presentation/model/dashboard_routes.dart';
 import 'package:tmail_ui_user/features/search/presentation/search_email_view.dart';
 import 'package:tmail_ui_user/features/thread/presentation/thread_view.dart';
 
 class MailboxDashBoardView extends BaseMailboxDashBoardView {
-
   MailboxDashBoardView({Key? key}) : super(key: key);
 
+  final mailBoxDashBoardController = Get.find<MailboxDashBoardController>();
```
Comment (dab246) [inline lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart:16]: No need to `Get.find`. you can directly use the `controller` object which is an instant of MailboxDashBoardController

File: lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart [hunk 2]
```diff
@@ -5,16 +5,49 @@ import 'package:get/get.dart';
...
+                child: MailboxView(),
+                width: ResponsiveUtils.defaultSizeLeftMenuMobile),
+            desktop: SizedBox(
+                child: MailboxView(),
+                width: ResponsiveUtils.defaultSizeLeftMenuMobile)),
+        drawerEnableOpenDragGesture:
+            responsiveUtils.hasLeftMenuDrawerActive(context) &&
+                !(mailBoxDashBoardController.dashboardRoute.value ==
+                    DashboardRoutes.emailDetailed),
+        body: Stack(children:[
```
Comment (dab246) [inline lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart:43]: why use `stack`

File: lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart [hunk 3]
```diff
@@ -5,16 +5,49 @@ import 'package:get/get.dart';
...
+                width: ResponsiveUtils.defaultSizeLeftMenuMobile),
+            tabletLarge: SizedBox(
+                child: MailboxView(),
+                width: ResponsiveUtils.defaultSizeLeftMenuMobile),
+            desktop: SizedBox(
+                child: MailboxView(),
+                width: ResponsiveUtils.defaultSizeLeftMenuMobile)),
+        drawerEnableOpenDragGesture:
+            responsiveUtils.hasLeftMenuDrawerActive(context) &&
+                !(mailBoxDashBoardController.dashboardRoute.value ==
```
Comment (dab246) [inline lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart:41]: 1. Please double check the email search case.
2. Test it on a web browser when resizing the screen.

File: lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart (hunk 2 above)
Comment (sherlockvn) [reply lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart:43]: I don't know. I don't add it, i already be there

**Review by ManhNTX — APPROVED** [review]
LGTM

File: lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart (hunk 2 above)
Comment (dab246) [reply lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart:43]: If so please delete it, it's really not needed now.

File: lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart (hunk 3 above)
Comment (hoangdat) [reply lib/features/mailbox_dashboard/presentation/mailbox_dashboard_view.dart]: please provide demo @sherlockvn

**Comment by hoangdat** [comment]
please provide demo for responsive in web

**Review by dab246 — APPROVED** [review]
/lgtm

**Pull request merged**